benchmark/benchmark_snn_single_gpu.py

```
@@ -0,0 +1,831 @@
+                            command.append("--channels-last")
+                        if args.require_gpu_name:
+                            command.extend(
+                                ["--require-gpu-name", args.require_gpu_name]
+                            )
+                        env = os.environ.copy()
+                        env["SJ_BENCH_SOURCE_ROOT"] = str(source_root)
+                        env["PYTHONPATH"] = str(source_root)
+                        env["TORCHINDUCTOR_CACHE_DIR"] = str(
+                            output_dir / "cache" / stem
+                        )
+                        subprocess.run(command, check=True, env=env)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)